ds-css/dist/index.html

```diff
@@ -3,9 +3,9 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title> CBP-DS THEME TEST SITE </title>
-    <link rel="stylesheet" href="css/main.css">
-  <link href="css/main.bundle.css" rel="stylesheet"></head>
+    <title> CBP-DS Theme Website </title>
+    <!-- CSS Files Below -->
```

Comment:
lets make sure the name of the css we distribute is cbp-ds.css for non minified and cbp-ds.min.css for minified.  Also, not sure we need the cache busting just yet.

Reply:
ok, made these updates and will send up again soon